op-e2e/celo/src/withdraw.js

```diff
@@ -1,17 +1,16 @@
-export const withdraw = async function (args, config) {
+export const withdraw = async function(args, config) {
   const initiateHash = await config.client.l2.wallet.initiateWithdrawal({
     request: {
       gas: args.gas,
       to: args.to,
       value: args.amount,
     },
-  })
+  });
   const receipt = await config.client.l2.public.waitForTransactionReceipt({
     hash: initiateHash,
-  })
+  });
 
-  const l2GasPayment =
-    receipt.gasUsed * receipt.effectiveGasPrice + receipt.l1Fee
+  const l2GasPayment = receipt.gasUsed * receipt.effectiveGasPrice;
```

Comment:
I thought we want to keep the `l1Fee`, just have it `0` everywhere, maybe doesn't make sense to remove it here.

Reply:
True, but it seems this is not yet reflected in the code. This means that receipt values do not align with deducted values. 
This only happens when the `blobbasefeeScalar` is **not** set to 0.

I [opened an issue for it](https://github.com/celo-org/op-geth/issues/255), I don't know if we were aware of this behavior already.

A hot-fix to make the test pass would be to set the `blobbasefeeScalar` value to 0 in the devnet deploy-config - currently the default values are used.

Reply:
@karlb reverted the [change](https://github.com/celo-org/op-geth/pull/253) (and merged only today), so we shouldn't be shortcutting anymore.

Reply:
The change is in op-geth and we didn't update the dependency yet. I'll probably do it as part of the optimism repo rebase.